packages/react-graphql/src/links.ts

```diff
@@ -52,18 +51,3 @@ class SsrExtractableLink extends ApolloLink {
     });
   }
 }
-
-export class ApolloClient<CacheShape> extends BaseApolloClient<CacheShape> {
```

Comment:
Recently we had been going with the apollo vs apollo-boost approach.
1. Providing a simple base usage
2. Export individual pieces to allow of future customization

for that I wonder if we want to keep this exported as a simple version of `createClient` that the consumer can choose to pass it into the self serializer themselves.